zcash_primitives/src/transaction/fees/zip317.rs

```diff
@@ -19,6 +19,7 @@ use crate::{
 ///
 /// [ZIP 317]: https//zips.z.cash/zip-0317
 pub const MINIMUM_FEE: Amount = Amount::const_from_i64(10_000);
+pub const MARGINAL_FEE: Amount = Amount::const_from_i64(5_000);
```

Comment:
Remember that each public item must be separately documented.

```suggestion

/// The marginal fee using the standard [ZIP 317] constants. Equivalent to
/// `FeeRule::standard().marginal_fee()`, but as a constant.
///
/// [ZIP 317]: https//zips.z.cash/zip-0317
pub const MARGINAL_FEE: Amount = Amount::const_from_i64(5_000);
```

Reply:
Oh hey, I wanted to make this change just yesterday!